### PR TITLE
Athera's circleCI conf

### PR DESCRIPTION
**Pull request by Tchomp42**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+          - ./artifacts/
+  upload_them_all:
+    working_directory: /go/src/github.com/athera-io/gcs-bucket-helper
+    docker:
+      - image: cibuilds/github:0.10
+    steps:
+      - attach_workspace:
+          at: .
+      - run:
+          name: "Gets GHR and uploads artifacts to Github"
```
Comment (sipickles) [inline .circleci/config.yml]: What is ghr?

Comment (Tchomp42) [reply .circleci/config.yml]: https://github.com/tcnksm/ghr

Comment (sipickles) [reply .circleci/config.yml]: It should be mentioned in the commit message. In fact, this commit has no message at all, which is not helpful.

Why do you want to do this change?

Comment (Tchomp42) [reply .circleci/config.yml]: @sipickles "Created CI conf file to build and upload binaries to the github repo"

Comment (sipickles) [reply .circleci/config.yml]: This should be in the commit message, not just a comment. Thats my point :)

**Pull request merged**